airbyte-integrations/connectors/destination-bigquery/build.gradle

```diff
@@ -1,5 +1,7 @@
 plugins {
     id 'airbyte-java-connector'
+    id "io.airbyte.gradle.docker"
+    id 'airbyte-connector-docker-convention'
```

Comment:
actually derp, do you need this in source-e2e-test also?

Reply:
I already did this in a previous PR: https://github.com/airbytehq/airbyte/commits/master/